README.md

```diff
@@ -13,24 +13,51 @@ Feel free to email [dev.careers@waveapps.com](dev.careers@waveapps.com) if you h
 
 ## Project Description
 
-Imagine that this is the early days of Wave's history, and that we are prototyping a new invoicing system in Canada. Our prototype will be entirely API-driven and use a Javascript single-page app as the interface to our APIs.
+### Background:
+
+Imagine that this is the early days of Wave's history, and that **we are prototyping a new invoicing system** in Canada. Our users are small business owners who need to generate professional-looking invoices to send to their customers. For this code challenge, build out **the settings page where users (business owners) can manage their customers' information**. Our prototype will be entirely API-driven and use a Javascript single-page app as the interface to our APIs.
+
+### Functional requirements:
+
+Customer information page should do the following:
+
+- Retrieve the data from the GET endpoint: https://cdn.jsdelivr.net/gh/wvchallenges/se-exp-challenge-invoice@master/settings.json
```

Comment:
Can you use the prior link for this? It needs to be updated with the Pages link from the prior commit